persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/ConversationDAO.kt

```diff
@@ -63,6 +65,32 @@ data class ConversationEntity(
     }
 }
 
+@Suppress("FunctionParameterNaming")
+data class ConversationViewEntity(
+    val id: QualifiedIDEntity,
+    val name: String?,
```

Comment:
Should ever a conversation have an empty name?

Reply:
1:1 conversations have no name, actually.

From my side I see your name, from your side you see mine, for example. We should always use the `OtherUser.name` as the conversation name.

Reply:
yes, but this field is the local entity field specific to each user's setup, so there is nothing stopping us to declare this `name` field as not nullable and map it with the appropriate name